C_CPP/Data Structures/Matrix/Diagonal_Matrix.md

```diff
@@ -0,0 +1,68 @@
+# Diagonal Matrix
+
+A diagonal matrix is a matrix in which the entries outside the main diagonal are all zero.Elements of the main diagonal can either be zero or nonzero .An example of a 3×3 diagonal matrix is:
+
+![diagonal matrix](https://imgur.com/rSR0Frx.jpg)
+
+Here , A[i][j]=0 (if i not equal to j)
```

Comment:
yes sure

Reply:
> kajal, you haven't mentioned what is i & j here or above it, I will recommend changing it to a[row][column], it will be easier to understand.

Made the changes